__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest")
+const app = require("../db/app")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data")
+const db = require("../db/connection")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe("/api", () => {
+    describe("GET", () => {
+        describe("/topics", () => {
+            test("all results should contain correct keys", () => {
+                return request(app)
+                .get("/api/topics")
+                .expect(200)
+                .then(({body}) => {
+                    const { topics } = body
+                    topics.forEach(topic => {
```

Comment:
if you're iterating through an array in your tests, you must include a line in your test to check that the array isn't empty (eg. checking the length isn't 0), otherwise this test will give you a false positive